src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Added new feature to detect secrets related information from Azure PowerShell cmdlet output with warning message. Learn more at https://go.microsoft.com/fwlink/?linkid=2258844
```

Comment:
```suggestion
* Added a preview feature to detect secrets and sensitive information from the output of Azure PowerShell cmdlets to prevent leakage. Enable it by `Set-AzConfig -DisplaySecretsWarning $true`. Learn more at https://go.microsoft.com/fwlink/?linkid=2258844
```